src/energyplus/ForwardTranslator/ForwardTranslateCoilSystemCoolingDXHeatExchangerAssisted.cpp

```diff
@@ -120,8 +125,11 @@ namespace energyplus {
         } else if (idf->iddObject().type() == IddObjectType::Coil_Cooling_DX_VariableSpeed) {
           idf->setString(Coil_Cooling_DX_VariableSpeedFields::IndoorAirInletNodeName, hxSupplyAirOutletNodeName);
           idf->setString(Coil_Cooling_DX_VariableSpeedFields::IndoorAirOutletNodeName, hxExhaustAirInletNodeName);
+        } else if (idf->iddObject().type() == IddObjectType::Coil_Cooling_DX) {
+          idf->setString(Coil_Cooling_DXFields::EvaporatorInletNodeName, hxSupplyAirOutletNodeName);
+          idf->setString(Coil_Cooling_DXFields::EvaporatorOutletNodeName, hxExhaustAirInletNodeName);
```

Comment:
Need to add FT to support Coil:Cooling:DX on CoilSystem:Cooling:DX:HeatExchangerAssisted.